enterprise/utils.py

```diff
@@ -542,9 +545,12 @@ def serialize_notification_content(
 
     email_items = []
     for user in users:
-        is_pending_user = hasattr(user, 'user_email') and not hasattr(user, 'first_name')
-        login_or_register = 'register' if is_pending_user else 'login'
-        destination_url = destination_url.format(login_or_register=login_or_register)
+        login_or_register = 'register' if is_pending_user(user) else 'login'
+        destination_url = '{site}/{login_or_register}?next={course_path}'.format(
+            site=lms_root_url,
+            login_or_register=login_or_register,
```

Comment:
Nice, and the refactor actually cleans up that weird format thing in the old code.